docs/mrt/api/main2.rst

```diff
@@ -0,0 +1,21 @@
+.. _mrt_main2_api:
+
+*************
+MRT main2 API
+*************
+
```

Comment:
Here you should give a brief for the logic of MRT quantization stage with corresponding codes.

Reply:
Already updated.